webclient/nginx.conf

```diff
@@ -78,10 +78,17 @@ http {
     # metadata
     location = /robots.txt { access_log off; }
     location = /googlebef9161f1176c5e0.html { access_log off; } # google search console
+    location = /sitemap-webclient.xml { access_log off; }  # webclient sitemap
     location = /favicon.ico { root /app/assets/;}
     # These Files are intenitonally not supported
     location = /adds.txt { log_not_found off; access_log off; }
     location = /app-ads.txt { log_not_found off; access_log off; }
+    
+    # sitemaps hosted on the cdn (webclient sitemap already catched above)
+    location ~ ^/sitemap {
+      access_log off;
+      proxy_pass https://nav.tum.sexy/cdn$request_uri;
```

Comment:
Wouldnt it be more efficient to directly link to the cdn? Why do we have to add this extra latency?

Reply:
My main concern was search engine bots not liking that redirect, because extra latency is not an issue for sitemaps. Though a redirect feels cleaner.

Reply:
No, I did not say a redirect, 
I said directly linking to them :)
Google does not require that the sitemaps are all under the root URL, or do they?

Reply:
Let's just try